addons/text_identification/test/EventSendingTests.js

```diff
@@ -36,5 +42,15 @@ TestCase("[Text Identification] Event sending", {
         this.presenter.triggerSelectionChangeEvent();
 
         assertFalse(this.eventBus.sendEvent.called);
+    },
+
+    'test GSA event calls showAnswers method' : function() {
+        var eventName = "GradualShowAnswers";
+        var eventData = {
+            moduleID: 'TextIdent1'
+        };
+        this.presenter.onEventReceived(eventName, eventData);
+
+        assertEquals(1, this.stubs.showAnswers.callCount);
     }
 });
```

Comment:
Brak nowej lini na koniec, (https://stackoverflow.com/questions/16222530/why-do-i-need-vim-in-binary-mode-for-noeol-to-work)

Dodałbym na pewno trochę testów sprawdzających czy odpowiednie funkcje się wykonały. 
Na przykład w zależności od 
presenter.configuration.shouldBeSelected
funkcja: applySelectionStyleShowAnswers zostanie wywołana albo nie. W przypadku przyjścia eventu GSA.

Możesz się sugerować innymi testami GSA w innych addonach, ponieważ takie sprawdzenia już są robione